module-apps/application-messages/data/SMSdata.hpp

```diff
@@ -81,8 +84,17 @@ class SMSSendTemplateRequest : public SMSRequest
         return preventAutoLock;
     }
 
+    [[nodiscard]] auto getNameOfSenderApp() const -> std::optional<app::ApplicationName>
+    {
+        if (nameOfSenderApp.has_value()) {
```

Comment:
This whole `if` could perhaps be just a `return`: if an optional holds no value, then an empty optional is returned anyway.